api/lib/infrastructure/repositories/sessions/session-for-supervising-repository.js

```diff
@@ -32,7 +45,7 @@ module.exports = {
 
 function _toDomain(results) {
   const certificationCandidates = results.certificationCandidates
-    .filter((candidate) => candidate !== null)
+    .filter((candidate) => candidate?.id !== null)
     .map((candidate) => new CertificationCandidateForSupervising({ ...candidate }));
```

Comment:
```suggestion
    .map((candidate) => new CertificationCandidateForSupervising(candidate));
```

Ça marcherait aussi non ?

Reply:
Oui tout à fait, c'etait pour expliciter qu'il ne s'attend pas à cet objet particulier (mais c'est une remarque que j'aurais pu faire 😄)

Reply:
ok, je laisse comme ça alors